operator/controllers/kyma_controller.go

```diff
@@ -252,15 +276,15 @@ func (r *KymaReconciler) HandleDeletingState(ctx context.Context, kyma *operator
 
 	if kyma.Spec.Sync.Enabled {
 		if err := remote.RemoveFinalizerFromRemoteKyma(ctx, r, kyma); client.IgnoreNotFound(err) != nil {
-			return false, err
+			return false, errwrap.Wrap(err, "error while trying to remove finalizer from remote")
```

Comment:
`errors.Wrap()` / `errors.Wrapf()` is obsolete. we could use `fmt.Errorf("your error string %w", err)` or `errors.Errorf()` for preserving strack trace

Reply:
Fixed it by replacing all Wraps with fmt.Errorf (as this is also used in the k8s libs)